### DISCUSSION TITLE
Apple Thunderbolt Display faded

### DISCUSSION THREAD

**Discussion by Director100DL** [MonitorControl/MonitorControl#781] [Q&A] [answered]
Hello
I have installed MonitorControl 4.0.1 on an iMac late 2012 running Catalina.
When I opened MonitorControl, my Apple Thunderbolt Display faded at 25% brightness event though the sliding bar on the menu indicates 100% in brightness value. I've uninstalled MonitorControl, but my Apple Thunderbolt Display still faded at 25%.
What can I do?

**Comment by waydabber** [comment, marked as answer]
Hi,

if your Thunderbolt display is not supported using the Apple protocol anymore, you can still control it via DDC. The display is rather non-standard in this regard, you need to manually override the max brightness setting as seen on this screenshot:

<img width="842" alt="Screen Shot 2021-10-26 at 18 42 13" src="https://user-images.githubusercontent.com/37590873/138923583-411f17b9-994c-4de7-b03d-0a8c525205d8.png">

Without this there is no way to fix the display as it has no buttons to change the brightness (Apple hates buttons!).

Hope this fixes the issue!

**Comment by Director100DL** [comment]
Hi,

Unfortunately, your solution didn’t fixed my problem.
I’ve installed version 4.0.2 today that seems buggy.
When I slide the brightness control in the menu bar, the movement of pointer of my mouse becomes clunky et freezes.


Denis Lévesque
Réalisateur

(418) 951-1381
***@***.***

**Reply by waydabber** [reply]
Well, that is new. I have no answer to that, I recommend using 4.0.1 or 4.0.0 then (the was the menu bar works was tweaked a bit in 4.0.1 but that should not cause a clunkiness issue like this).

I am sorry about the Thunderbolt display - some users report changing the DDC value works, some don't, maybe there are differences in terms of hardware revision of firmware. Some users reported success in changing the brightness via ddcctl on the display. The difference might also related to how aggressive DDC timeouts/number of retries the app uses during communication.
